app/src/main/res/layout/fragment_welcome.xml

```diff
@@ -1,63 +1,61 @@
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context="org.fossasia.openevent.general.WelcomeFragment">
 
-    <LinearLayout
-        android:id="@+id/textLinearLayout"
+    <RelativeLayout
```

Comment:
Whats the need of RelativeLayout

Reply:
For setting up the ImageView between text LinearLayout and the Button to look UI better for all sizes device.

Reply:
Screenshot please

Reply:
same as earlier - 
<p align="center">
<img src="https://user-images.githubusercontent.com/27783786/51200703-4479ed80-1920-11e9-89d3-4309d5fc5d07.png" height = "480" width="270"> 
</p>

Reply:
RealtiveLayout is a little bit expensive than LinearLayout, so it should be avoided. Here the layout is simple, so I'll approve